src/MyPlot/MyPlot.php

```diff
@@ -898,6 +898,7 @@ public function buyPlot(Plot $plot, Player $player) : bool {
 		if($ev->isCancelled()) {
 			return false;
 		}
+		$this->getEconomyProvider()->addMoney($this->getServer()->getOfflinePlayer($plot->owner), $plot->price);
```

Comment:
make sure to check if the payment fails. We don't want the plot transferred if the owner is never paid.